packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -241,6 +241,9 @@ export const prepareDiagnosticMetricItem = (webex: any, item: any) => {
   const cdl = webex.internal.newMetrics.callDiagnosticLatencies;
 
   switch (eventName) {
+    case 'client.webexapp.launched':
```

Comment:
Case options should be constants in this context. Would these be available via `@webex/event-dictionary-ts` within the `json` scope or should we simply create a "white-list" of constants we use?

Note that we could also generate another artifact from event-dictionary that could include just "event names" for consistency.

Reply:
these are typed, so no need for constants

Reply:
still feel this need to be constant for 
single place to visualize event accoss the board , easy to maintain test and actual src constants

Reply:
This makes sense, we don't want to "thicken" the bundle size by importing anything material from `@webex/event-dictionary-ts`. But, it would be helpful to have a parody of this locally:

```ts
/* ...constants.ts... */

const EVENT_NAMES: Record<string, MetricEventNames> = {
  EVENT_NAME_A: 'some-metrics-event-name',
};
```

These can then be used as a whitelist for what is currently consumed by this plugin without having to search the plugin file-by-file.

Reply:
Typescript already guarantees that the event name used here is correct, if you do a typo in it, it will flag an error. I see no added value in making a constant or enum that you then have to click and jump to another file to see the value and when you search for it your first result is not in the file where it's actually used and you have to do another search for the constant to find the code. 
We normally use constants to avoid magic values like 1 or 10 in the code that can have many meanings and a constant guarantees that you know which "1" you want to use in case you ever want to change it. Here, these values like 'client.webexapp.launched' are unique and it's not like you will see the same string 'client.webexapp.launched' somewhere else that would mean a different thing.